site/docs/v1/tech/apis/jwt.adoc

```diff
@@ -481,6 +481,8 @@ include::docs/v1/tech/apis/_standard-delete-response-codes.adoc[]
 
 This API is used to validate a JSON Web Token. A valid JWT indicates the signature is valid and the payload has not be tampered with and the token is not expired.
 
+You can also validate a JWT without using this API call. You can validate the JWT attributes and signature locally. Most programming languages have libraries to help with this. Here's an https://github.com/fusionauth/fusionauth-jwt#verify-and-decode-a-jwt-using-hmac[example in Java]. Unless an HMAC key was used to sign the JWT and you don't have access to the key, it is recommended to validate the JWT locally.
```

Comment:
Do you think we should be making this recommendation? I would think of these are options, and we could outline pros and cons, but the choice will be up to the user.